codeclimate-cppcheck.py

```diff
@@ -158,23 +176,27 @@ def cppcheck_error_to_codeclimate_issue(error):
     issue['categories'] = [category]
 
     issue['location'] = convert_location(error[0])
+    issue['other_locations'] = []
     if len(error) > 1:
         locations = list(error)[1:]
-        locations = [convert_location(l) for l in locations]
-        issue['other_locations'] = locations
+        for l in locations:
+            if (l.get('line') is not None):
```

Comment:
Parentheses are not needed here.

Reply:
I fixed this myself and landed as https://github.com/antiagainst/codeclimate-cppcheck/commit/f6d8acfe9b60c5add15ffdd3446c16e8e513bb78.